File: 2 semestre/Laboratório de Desenvolvimento de Algoritmos/Lista de Exercícios 3/src/Ex11.java
//Importando Scanner
import java.util.Scanner;

public class Ex11 {
	
	public static void main(String[] args){
		
		//Instanciando Scanner
		Scanner ent = new Scanner(System.in);
		
		System.out.println("Soma de pares e ímpares\n\n");
		
		//vars
		int somaPar=0, somaImpar=0, num;
		
		//Entrada
		System.out.print("Digite um número menor ou igual a 1000, ou 1000 para terminar: ");
		num = ent.nextInt();
		
		while ( num <= 1000 )
		{
			//Processamento
			if ( num%2 ==0 )
				somaPar += num;
			else
				somaImpar += num;
			
			//Entrada
			System.out.print("Digite um número menor ou igual a 1000, ou 1000 para terminar: ");
			num = ent.nextInt();
			
		}
		
		//Saída
		System.out.println("\nSoma dos pares: " + somaPar);
		System.out.println("Soma dos ímpares: " + somaImpar);
		
		
	}
}

File: 2 semestre/Técnica de Desenvolvimento de Algoritmos/Listas de Exercicios/Lista 3/ex11.c
#include <stdio.h>

main()
{
    printf("Soma dos Pares e Impares\n\n");

    //vars
    int num, somaPar=0, somaImpar=0;

    //Entrada
    printf("Insira um numero ate 1000, ou maior que 1000 para terminar: ");
    scanf("%d",&num);

    while(num <= 1000)
    {
        if( num%2 == 0)
            somaPar += num;
        else
            somaImpar += num;

        printf("Insira um numero ate 1000, ou maior que 1000 para terminar: ");
        scanf("%i",&num);
    }

    printf("\nSoma dos pares: %i\n", somaPar);
    printf("Soma dos impares: %i", somaImpar);



}
